src/NameFieldSet.tsx

```diff
@@ -59,6 +59,32 @@ export class NamesFieldSet extends React.Component<INameFieldSetProps, INameFiel
     this.setState({ names: [...this.state.names, ""] });
   }
 
+  private arrayContainsDuplicates = (a: string[]) => {
+    const uniqueSet:Set<string> = new Set();
+    for(let i = 0; i <= a.length; i++) {
+      if (a[i] == null) {
```

Comment:
why would we have an name as null? is this even possible?

Reply:
`== null` _also_ catches undefined (which is possible): https://stackoverflow.com/questions/2647867/how-to-determine-if-variable-is-undefined-or-null